wtxmgr/tx.go

```diff
@@ -935,3 +959,76 @@ func (s *Store) Balance(ns walletdb.ReadBucket, minConf int32, syncHeight int32)
 
 	return bal, nil
 }
+
+// PutTxLabel validates transaction labels and writes them to disk if they
+// are non-zero and within the label length limit.
+func (s *Store) PutTxLabel(ns walletdb.ReadWriteBucket, txid chainhash.Hash,
+	label string) error {
+
+	return putTxLabel(ns, txid, label)
+}
+
+// putTxLabel writes a transaction label to disk, keyed by transaction hash:
+// [0:32] Transaction hash (32 bytes)
+//
+// The label itself is written to disk in length value format:
+// [0:2] Label length
+// [2: +len] Label
+func putTxLabel(ns walletdb.ReadWriteBucket, txid chainhash.Hash,
```

Comment:
Nit: doesn't seem like we need an exported and unexported version of this method? Just having the exported would be fine.